department-of-veterans-affairs/caseflow
# Schedule add null state to modal

Resolves #8368

### Description


### Acceptance Criteria
- [ ] Add text to the null state for selectors, "Please Select..."

### Testing Plan
1. Go to Daily Docket.
2. Click `Edit Hearing Day`  modal
3. Verify `Please Select...` is the placeholder.
Ps.
I removed the empty object that was showing whenever user selected.

<img width="496" alt="screen shot 2018-12-21 at 4 57 34 pm" src="https://user-images.githubusercontent.com/23080951/50365288-c9e8c880-0541-11e9-8472-4b67c138b83e.png">
